build-system/tasks/make-extension/index.js

```diff
@@ -24,304 +24,183 @@ const {
 const {cyan, green, red, yellow} = require('kleur/colors');
 const {execOrThrow} = require('../../common/exec');
 const {log} = require('../../common/logging');
-const {makeBentoExtension} = require('./bento');
 
-const year = new Date().getFullYear();
-
-/*eslint "max-len": 0*/
 /**
- * @param {string} str
+ * Convert dash-case-name to PascalCaseName.
+ * @param {string} name
  * @return {string}
  */
-function pascalCase(str) {
-  return (
-    str[0].toUpperCase() +
-    str.slice(1).replace(/-([a-z])/g, function (g) {
-      return g[1].toUpperCase();
-    })
-  );
+function dashToPascalCase(name) {
```

Comment:
suggestion: move this to `src/core/types/string.js` where `dahToCamelCase` and others live

Reply:
Cannot import `src/` files onto `build-system` since `src` is `esm` and our node files are `cjs`.

```
/Users/alanorozco/git/amphtml/src/core/types/string.js:39
export function dashToCamelCase(name) {
^^^^^^

SyntaxError: Unexpected token 'export'
```